packages/babel-preset/index.js

```diff
@@ -114,5 +100,17 @@ module.exports = function shopifyCommonPreset(
       require.resolve('@babel/plugin-transform-react-constant-elements'),
   ].filter(Boolean);
 
-  return {presets, plugins};
+  // proposal-class-properties, plugin-proposal-private-methods must be used in loose mode (this is now handled by these assumptions)
```

Comment:
We can now use assumptions for our legacy decorator plugins. Updated the comments to reflect this.